.github/workflows/CI.yml

```diff
@@ -246,6 +246,16 @@ jobs:
         # * copy only main/first project binary
         project_exe_stem=${PROJECT_EXES%% *}
         cp "src/${project_exe_stem}${{ steps.vars.outputs.EXE_suffix }}" "${{ steps.vars.outputs.PKG_DIR }}/bin/${project_exe_stem}-gui${{ steps.vars.outputs.EXE_suffix }}"
+        # create and stage text+gui hybrid for Windows
+        case '${{ matrix.job.os }}' in windows-*)
+          # * clean/remove build artifact(s)
+          opam exec -- make -C src clean ## .or.# opam exec -- make clean #.or.# rm "src/${project_exe_stem}${{ steps.vars.outputs.EXE_suffix }}"
```

Comment:
I think `rm "src/${project_exe_stem}${{ steps.vars.outputs.EXE_suffix }}"` is plenty sufficient here. Just to avoid the entire rebuild. (The designation as console or GUI application happens at link time, so rebuild is not needed.)

Reply:
Yes, since it's only a link step change, I thought that myself. But I wasn't completely sure, so I implemented it as a half measure of rebuild. With just the 'src' clean, the recompilation literally completes within seconds and doesn't change the total build time in any real way, so I left it.

I'll change to the `rm ...` and re-push if you think its warranted.

Reply:
Thinking a bit more about it, perhaps a more relevant improvement is to make this new code a separate job step guarded by an `if: runner.os == 'Windows'`. I think this makes the build log easier to follow should anything fail. It also makes the scripts simpler (trying to go for a universal build script with GTK on Win is kind of futile at this point, something we have to live with).